File: examples/holidays/countries/node/listCountries.js
const { HolidaysService } = require("m3o/holidays");

// Get the list of countries that are supported by this API
async function listCountries() {
  let holidaysService = new HolidaysService(process.env.MICRO_API_TOKEN);
  let rsp = await holidaysService.countries({});
  console.log(rsp);
}

listCountries();

File: examples/go/holidays/countries/listCountries.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/holidays"
)

// Get the list of countries that are supported by this API
func ListCountries() {
	holidaysService := holidays.NewHolidaysService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := holidaysService.Countries(&holidays.CountriesRequest{})
	fmt.Println(rsp, err)
}
